src/tests/unit/test-utils.js

```diff
@@ -27,21 +35,18 @@ const resourceSchema = (resourceType, resourceId, resourceAttributes) => {
     },
   };
   if (resourceAttributes) {
-    resourceAttributes = _.mapKeys(resourceAttributes,
-      (value, key) => _.camelCase(key));
-    schema.data.attributes = resourceAttributes;
+    // populate object of attributes defined by swagger
+    schema.data.attributes = _.reduce(_.keys(getDefinition('Attributes')), (result, value) => {
+      result[value] = null;
+      if (resourceAttributes[value]) {
+        result[value] = resourceAttributes[value];
+      }
+      return result;
```

Comment:
I think an if else would work better here. Or a ternary conditional.
`result[value] = resourceAttributes[value] ? resourceAttributes[value] : null`

Do you actually need the conditional? Wouldn't `resourceAttributes[value]` either return null or a valid value?

Reply:
This conditional is still needed because the serializer differentiates between undefined and null. Just saying `result[value] = resourceAttributes[value]` would result in the serializer setting attributes to undefined, not null. I'll still shorten up this statement with a ternary condion